bika/lims/browser/analysisrequest/analysisrequests.py

```diff
@@ -124,12 +124,12 @@ def __init__(self, context, request):
                 "attr": "getSampleID",
                 "index": "getSampleID",
                 "replace_url": "getSampleURL",
-                "toggle": True, }),
+                "toggle": False}),
             ("BatchID", {
                 "title": _("Batch ID"),
                 "index": "getBatchID",
                 "sortable": True,
-                "toggle": True}),
+                "toggle": False}),
```

Comment:
Mental note for @xispa. If we set this field as "hidden" by default, then we need to set visible by default in health add-on, cause in that add-on, Batches are overrided as Clinical cases.